pkg/ucp/integrationtests/planes/testdata/awsplane_v20231001preview_list_responsebody.json

```diff
@@ -1,12 +1,14 @@
 {
-    "values": [
```

Comment:
I knew I saw **values** somewhere. You changed it. Thank you!

Reply:
Examples are pretty frequently wrong, because they are manual 😢